packages/babel-cli/package.json

```diff
@@ -29,7 +29,8 @@
     "source-map": "^0.5.0"
   },
   "optionalDependencies": {
-    "chokidar": "^2.1.8"
+    "chokidar": "^3.4.0",
+    "chokidarAt2": "https://registry.npmjs.org/chokidar/-/chokidar-2.1.8.tgz"
```

Comment:
Note that it may cause trouble for projects a) developed in where `registry.npmjs.org` may not be accessible and b) using Node.js 6. In that case, please upgrade local Node.js versions. I leave a comment here for future reference.